tests/load/utils.py

```diff
@@ -606,7 +598,7 @@ def destinations_configs(
                     bucket_url=bucket,
                     extra_info=bucket,
                     supports_merge=False,
-                    file_format="parquet",
+                    file_format="jsonl",  # keep jsonl as default, test utils are setup for this
```

Comment:
Note for review: I changed the default format to jsonl so we save more core in the test utils for the sftp special case, I didn't see any downside to this.